machine_translation/prepare_data.py

```diff
@@ -157,11 +160,7 @@ def merge_parallel(src_filename, trg_filename, merged_filename):
     with open(src_filename, 'r') as left:
         with open(trg_filename, 'r') as right:
             with open(merged_filename, 'w') as final:
-                while True:
-                    lline = left.readline()
-                    rline = right.readline()
-                    if (lline == '') or (rline == ''):
-                        break
+                for lline, rline in equizip(left, right):
                     if (lline != '\n') and (rline != '\n'):
```

Comment:
I think this is not necessary. The file iterators created because you use the `for` statement will get exhausted, and the iteration will stop. Can you please try?